server/builder/receiver.js

```diff
@@ -1,19 +1,17 @@
-const Queue = require('./queue')
+const Queue = require('../rabbitmq')
 const execFile = require('child_process').execFile
-const config = require('./config')
+const config = require('../src/config')
 const rp = require('request-promise')
 const fs = require('fs')
 
 receiver().then(console.log('receiver started'))
 
-// TODO: Bufferize the output to send 4096 char at a time to not send too much requests
-
 async function receiver () {
   const queue = new Queue('build-raven-os-org')
 
   await queue.receive(async (msg) => {
     const data = [...msg.content]
-    const url = config.build_api_url + 'manifest/'
+    const url = config.url + 'manifest/'
```

Comment:
Don't you have removed this configuration from `server/src/config/config.js` in this commit?

Reply:
builder folder removed from server folder. It was forgotten after the rebase.